### PR TITLE
raftkv: add comments.

### PR DESCRIPTION
**Pull request by ngaut**

### CODE REVIEW

File: src/storage/engine/raftkv.rs [hunk 1]
```diff
@@ -162,6 +162,7 @@ impl<T: PdClient, Trans: Transport> RaftKv<T, Trans> {
         cmd.set_header(header);
         cmd.set_requests(RepeatedField::from_vec(reqs));
         let resp = try!(self.exec_cmd_request(cmd));
+        // notice: raft thread would be wake up after the take method been called.
```
Comment (siddontang) [inline src/storage/engine/raftkv.rs]: I guess correct should be  "has been called"

Comment (BusyJay) [inline src/storage/engine/raftkv.rs]: s/wake/waken/

**Comment by BusyJay** [comment]
LGTM

File: src/storage/engine/raftkv.rs [hunk 2]
```diff
@@ -162,6 +162,7 @@ impl<T: PdClient, Trans: Transport> RaftKv<T, Trans> {
         cmd.set_header(header);
         cmd.set_requests(RepeatedField::from_vec(reqs));
         let resp = try!(self.exec_cmd_request(cmd));
+        // notice: raft thread would be woken up after the take method been called.
```
Comment (siddontang) [inline src/storage/engine/raftkv.rs]: s/been/has been/

**Comment by ngaut** [comment]
PTAL

**Comment by siddontang** [comment]
LGTM

**Pull request merged**